frontend/src/components/TransactionItem.tsx

```diff
@@ -212,7 +212,11 @@ function TransactionItem({ tx }: Props) {
               <p>Date & Time</p>
               <p className="text-muted-foreground">
                 {dayjs(tx.updatedAt)
-                  .tz(dayjs.tz.guess())
+                  .tz(
```

Comment:
Is this really the only place there is an issue despite using dayjs everywhere?

To me this feels like a hack. What about using `local`? https://day.js.org/docs/en/manipulate/local

Reply:
Yeah the problem is only with the `guess` function, updated to use `local` instead.